### PR TITLE
LibBigWig 0.3.0 and WiggleTools 1.2.1 (new formulas)

### PR DESCRIPTION
**Pull request by dzerbino**
Dear homebrew-science maintainers,

Please find attached two recipes for a tool that I developed [WiggleTools](https://github.com/Ensembl/WiggleTools) and a nifty library it depends on [libBigWig](https://github.com/dpryan79/libBigWig). 

Both repos have a few stars, so are hopefully of interest to the community. Both are also quite stable, as you can see from the relative absence of updates in the past few months. 

Thank you,

Daniel

### Have you:

- [Y] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [Y] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [Y] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [Y] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [Y] Written a sensible test? (The best test is to compile and run a sample program.)

### CODE REVIEW

**Comment by dzerbino** [comment]
Hmm, a couple of the error messages are confusing/unclear to me:
-  GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
Is that a deal breaker?
- Error: No formulae found in taps.
I thought I had just added it?
- Error: Testing requires the latest version of linuxbrew/science/[...]
Isn't that the point of a test?

- Non-executables were installed etc...
That's on me. I thought I had dealt with that bug by my last commit on WiggleTools, but maybe not...

**Comment by sjackman** [comment]
Hi, Daniel. This tap Linuxbrew/science is my own fork of Homebrew/science for the purpose of building binary bottles for Linux. Please submit this PR instead to Homebrew/science. Sorry for the confusion! That should resolve the middle two CI errors.

> `GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)`
Is that a deal breaker?

No, it's not a deal breaker. In Homebrew/science, publication is sufficient, and WiggleTools is published. http://bioinformatics.oxfordjournals.org/content/30/7/1008

File: libbigwig.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Libbigwig < Formula
+  desc "C library for handling bigWig files"
+  homepage "https://github.com/dpryan79/libBigWig"
+  url "https://github.com/dpryan79/libBigWig/archive/0.3.0.tar.gz"
+  sha256 "221002fe249e8099009f0790f44bfe991e85cb23763cf5fc494e745c0160edc2"
```
Comment (sjackman) [inline libbigwig.rb:5]: Please add the comment
```
# tag "bioinformatics"
```

File: wiggletools.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class Wiggletools < Formula
+  desc "Compute genome-wide statistics with composable iterators"
+  homepage "https://github.com/Ensembl/WiggleTools"
+  url "https://github.com/Ensembl/WiggleTools/archive/v1.2.1.tar.gz"
+  sha256 "906d32e281fe234b3eacbe569c21e68d61aca3d0ef2eec501e4efd61799be4ee"
```
Comment (sjackman) [inline wiggletools.rb:5]: Please add the comments
```
# doi "10.1093/bioinformatics/btt737"
# tag "bioinformatics"
```

**Comment by dzerbino** [comment]
OK, thanks for the detailed explanations!

**Pull request closed without merging**